docs/diagrams/ci/CodeCoverageReportingActivityDiagram.puml

```diff
@@ -0,0 +1,24 @@
+@startuml
```

Comment:
Same as above, if diagrams are included they should be embedded in the DG as images.